app/src/main/java/com/pileproject/drive/programming/visual/block/repetition/RepetitionBlock.java

```diff
@@ -29,14 +30,14 @@
  */
 public abstract class RepetitionBlock extends BlockBase {
 
-    public static final int FOREVER_WHILE_OFFSET = -1000;
+    static final int FOREVER_WHILE_OFFSET = -1000;
```

Comment:
Moved to ExecutionCondition, since this field is irrelevant for the operation of blocks.
(btw ExecutionCondition is no longer "condition")